### PR TITLE
android_studio.md: generate Android Studio releases list

### PR DESCRIPTION
**Pull request by hsz**

### CODE REVIEW

File: topics/products/android_studio.md [hunk 1]
```diff
@@ -20,13 +20,24 @@ Some minor modifications to the skeleton project are needed, as discussed below.
...
+
+In your Gradle build script, you should set both versions – build number and the release number – to the `intellij.version` property.
+To figure out the exact release number based on the build number, visit the [IntelliJ Repository Releases](https://www.jetbrains.com/intellij-repository/releases/) listing and check the `com.jetbrains.intellij.idea` section.
+
 The [build.gradle configuration steps](#configuring-the-plugin-buildgradle-file) section below explains how to set the IntelliJ Platform version to match the target version of Android Studio.
 
 ![Example Android Studio About Dialog](android_studio_build.png){width="600"}
 
+Below you may find a list of Android Studio releases mapped to the relevant IntelliJ IDEA versions:
+
```
Comment (YannCebron) [inline topics/products/android_studio.md:38]: add heading?

File: .github/workflows/generate-android-studio-releases.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  example-action:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Generate
+        run: .github/workflows/generate-android-studio-releases.sh
+      - name: Commit changes
+        uses: Endbug/add-and-commit@v8
+        with:
+          message: 'Generate Android Studio releases'
```
Comment (YannCebron) [inline .github/workflows/generate-android-studio-releases.yml:17]: include timestamp/date in message?

File: .github/scripts/android_studio_releases.main.kts [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+import org.json.JSONObject
+import org.simpleframework.xml.Attribute
+import org.simpleframework.xml.ElementList
+import org.simpleframework.xml.Path
+import org.simpleframework.xml.Root
+import org.simpleframework.xml.core.Persister
+import java.io.File
+import java.net.URL
+
```
Comment (YannCebron) [inline .github/scripts/android_studio_releases.main.kts:14]: please add one-line description of what this script does

File: .github/workflows/generate-android-studio-releases.yml (hunk 2 above)
Comment (hsz) [reply .github/workflows/generate-android-studio-releases.yml:17]: There's no need for that – commit will have one.

**Pull request merged**